pig/src/org/partiql/pig/cmdline/CommandLineParser.kt

```diff
@@ -47,6 +48,12 @@ class CommandLineParser {
         }
     }
 
+    private object PathValueConverter : ValueConverter<Path> {
+        override fun convert(value: String?): Path = Paths.get(value).toAbsolutePath().normalize()
```

Comment:
nit: my IntelliJ is giving a warning that `value` is nullable string but `Paths.get(...)` expects a non-null string

Reply:
I can remove the `?` but this doesn't do anything but make IntelliJ happy, since all it does is cause an NPE earlier.  This is a quirk related to Kotlin's interop with IntelliJ.